project/api/models.py

```diff
@@ -98,3 +99,49 @@ def clean(self, *args, **kwargs):  # pylint: disable=W0221
     def save(self, *args, **kwargs):  # pylint: disable=W0221
         self.full_clean()
         super(PoliticalRelation, self).save(*args, **kwargs)
+
+
+class AtomicPolygon(models.Model):
+    """
+    Stores geometric data corresponding to a wikidata ID
+    """
+
+    wikidata_id = models.PositiveIntegerField(primary_key=True)  # Excluding the Q
```

Comment:
What's the concern actually?

I wish to map it 1-1 with WD, if possible. Not sure if that's possible though: let's say an army won a battle an gained a small piece of land that day, not sure if wikidata has an entry for that piece of land.

Reply:
Yeah, I would imagine that we would run into situations like that somewhat frequently, but can't confirm. I think you'd know better. Maybe we should add a standard id field and keep a nullable and unique `wikidata_id` field?

Reply:
Yes, that sounds good. However for TEs, I would like a 1-1 mapping with wd_id. Do you see problems there too?

Reply:
I guess not. If they're missing data we'll have our mappers create a wikidata entity for it first. I was a bit worried about them lacking entities for some more obscure occupied territories originally, but again I haven't looked a ton into it.

It should be as long as it's not a CharField which stores empty strings as `""` rather than `NULL`. If it's not we can always just perform the validation as a pre-save hook I guess.